lib/bible_bot/bible.rb

```diff
@@ -570,11 +572,7 @@ def scripture_re
 
     def get_book_re_string
       # Get a regular expression string that will match any book of the Bible
-      r=''
-      Bible.books.each do |b|
-        r += sprintf('%s|', b.regex)
-      end
-      return r
+      Bible.books.map(&:regex).join('|')
```

Comment:
This was causing issues because as written, it was adding an `|` at the very end of the list of books, which ended up matching a lot of whitespace. We only want the `|` to be added between the list of books, so a simple `join` should do the trick.